### PR TITLE
CWL Support 1

### PR DESCRIPTION
**Pull request by jmchilton**
- Add project template to bring CWL draft 2 tools - ``planemo project_init --template cwl_draft2_spec``
 - Add ``--cwl`` and ``--cwl_galaxy_root`` options to ``serve`` command to serve subset CWL in Galaxy.

### CODE REVIEW

**Comment by jmchilton** [comment]
Going to merge soon, but thought I would do an afternoon ping in case anyone wanted to say their peace :).

File: README.rst [hunk 1]
```diff
@@ -218,6 +218,18 @@ For more information see the documentation for the `brew
 and `brew_env
 <http://planemo.readthedocs.org/en/latest/commands.html#brew_env-command>`__ commands.
 
+--------------------------
+Common Workflow Language
+--------------------------
+
+Planemo includes highly experimental support for running a subset of
```
Comment (martenson) [inline README.rst]: Maybe a mention that it runs modified Galaxy would be useful.

**Comment by martenson** [comment]
:+1:

**Comment by jmchilton** [comment]
@martenson Thanks, change made.

**Pull request merged**